modules/sr/robot/motor.py

```diff
@@ -76,15 +74,12 @@ def power(self, value: int) -> None:
 
         self.sr_motor.set_speed(translate(value, self.sr_motor))
 
-    ''''@property
-    def use_brake(self) -> bool:
-        "Whether to use the brake when at 0 speed"
-        return self._use_brake
+    @property
+    def use_brake(self):
+        """Whether to brake when motor speed is 0"""
+        return self.sr_motor._use_brake
 
     @use_brake.setter
-    def use_brake(self, value: bool) -> None:
-        self._use_brake = value
-
-        if self.power == 0:
-            "Implement the new braking setting"
-            self.power = 0'''
+    def use_brake(self, value):
```

Comment:
```suggestion
    def _use_brake(self, value):
```

As we are not encouraging students to use this function directly, it's best that we make it a 'private' method.